diff --git a/tests/base.py b/tests/base.py
index 35fcc3d..9566474 100644
--- a/tests/base.py
+++ b/tests/base.py
@@ -33,6 +33,7 @@ from xmldiff import main
 from xmldiff.actions import MoveNode
 
 from cyclonedx.output import SchemaVersion
+import cyclonedx
 
 if sys.version_info >= (3, 7):
     from jsonschema import ValidationError, validate as json_validate
@@ -45,7 +46,7 @@ else:
 cyclonedx_lib_name: str = 'cyclonedx-python-lib'
 cyclonedx_lib_version: str = version(cyclonedx_lib_name)
 single_uuid: str = 'urn:uuid:{}'.format(uuid4())
-schema_directory = os.path.join(os.path.dirname(__file__), '..', 'cyclonedx', 'schema')
+schema_directory = os.path.join(os.path.dirname(cyclonedx.__file__), 'schema')
 
 
 class BaseJsonTestCase(TestCase):
diff --git a/tests/test_spdx.py b/tests/test_spdx.py
index 6395f86..f8a5558 100644
--- a/tests/test_spdx.py
+++ b/tests/test_spdx.py
@@ -25,8 +25,9 @@ from unittest import TestCase
 from ddt import data, ddt, idata, unpack
 
 from cyclonedx import spdx
+import cyclonedx
 
-with open(path_join(dirname(__file__), '..', 'cyclonedx', 'schema', 'spdx.schema.json')) as spdx_schema:
+with open(path_join(dirname(cyclonedx.__file__), 'schema', 'spdx.schema.json')) as spdx_schema:
     KNOWN_SPDX_IDS = json_load(spdx_schema)['enum']
 
 VALID_COMPOUND_EXPRESSIONS = {
